openseeds/cohort-procedures-and-templates/speedblog.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
It could be good to provide a link to that page from the graduation guide

Reply:
Done. Thank you...